hybrid_a_star:
  search_phase:
    default_sim_time: 60
    precision_phase_distance: 1000
    precision_phase_sim_time: 20
  search_pruning:
    prune_radius_explored: 25
    prune_radius_closed: 25
  heuristic:
    voronoi_field_cost_weight: 0.005

mission_planner:
  #Name of mission, for path and debug data storage
  mission_name: "testmission"
  #If path determined in advance and stored to gpx, load it and publish
  load_from_gpx: false
  #Load from this gpx or save to this gpx
  gpx_name: "test_gpx"
  #Preprocessing has been run for mission region
  preprocessed_map: true
  #Preprocessed data to load (or save). Loads/saves collision db, voronoi db and quadtree 
  map_name: "outside_new_york"
  # Two coordinates defining the mission region. Discarded if preprocessed_map=true
  map_extent: [[1,2],[3,4]]
  #When vessel pose and goal received, immideately searches for path
  search_immideately: true


  
